pkg/analyze/velero.go

```diff
@@ -653,19 +653,22 @@ func getVeleroVersion(excludedFiles []string, findFiles getChildCollectedFileCon
 	veleroDeploymentDir := "cluster-resources/deployments"
 	veleroVersion := ""
 	veleroDeploymentGlob := filepath.Join(veleroDeploymentDir, "velero.json")
-	veleroDeploymentJson, err := findFiles(veleroDeploymentGlob, excludedFiles)
```

Comment:
shouldn't we still handle this error _as well as_ checking for zero length deployments?